teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml

```diff
@@ -70,3 +70,6 @@
           !include-raw:
             - ../../../scripts/build_utils.sh
             - ../../build/build
+
+    publishers:
+      - github-notifier
```

Comment:
Well, that's very confusing: it works now. I'm not sure why it didn't before, but unless it breaks again I won't worry about it.